### PR TITLE
Update _settings.scss

### PR DESCRIPTION
**Pull request by aqeelfikree**

### CODE REVIEW

File: core/neat/settings/_settings.scss [hunk 1]
```diff
@@ -47,7 +47,7 @@ $neat-grid: () !default;
 ///
 /// @property {number (with unit) | string | null} media [null]
 ///   The `@media` definition that is used by the [`grid-media`](#grid-media)
-///   mixin to detirmine the media properties.
+///   mixin to determine the media properties.
```
Comment (tysongach) [inline core/neat/settings/_settings.scss]: Looks like this was already fixed in https://github.com/thoughtbot/neat/commit/483d14341064420a6b90fcc8fc8dee2d2c2de57f.

Comment (whmii) [inline core/neat/settings/_settings.scss]: yep

File: core/neat/settings/_settings.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @charset "UTF-8";
 /// This variable is a sass map that overrides Neat's default grid settings.
-/// Use this to define your project's grid properties incluting gutters and
+/// Use this to define your project's grid properties including gutters and
```
Comment (tysongach) [inline core/neat/settings/_settings.scss]: @whmii This typo still remains, though.

**Comment by tysongach** [comment]
Thanks, @aqeelfikree! Merged as https://github.com/thoughtbot/neat/commit/5077b4b24fa24c4ce2cb05d841823a8ab9d43c36.

**Pull request closed without merging**